xchange-binance/src/main/java/org/knowm/xchange/binance/dto/account/AssetDividendList.java

```diff
@@ -0,0 +1,49 @@
+package org.knowm.xchange.binance.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import lombok.Data;
+
+public final class AssetDividendList extends SapiResponse<List<AssetDividendList.AssetDividend>> {
+
+  private final AssetDividend[] rows;
+
+  private final Long total;
+
+  public AssetDividendList(
+      @JsonProperty("rows") AssetDividend[] rows,
+      @JsonProperty("total") Long total) {
+    this.rows = rows;
+    this.total = total;
+  }
+
+  @Override
+  public List<AssetDividend> getData() {
+    return Arrays.asList(rows);
+  }
+
+  @Override
+  public String toString() {
+    return "AssetDividendList [rows=" + Arrays.toString(rows) + "]";
+  }
+
+  @Data
+  public static final class AssetDividend {
+
+    //    {
+    //           "amount":"10.00000000",
+    //            "asset":"BHFT",
+    //            "divTime":1563189166000,
+    //            "enInfo":"BHFT distribution",
+    //            "tranId":2968885920
+    //    },
```

Comment:
Comments

Reply:
Hello, see DepositList, we also have an example of response structure.
It could be useful. 

But i can remove it if needed.

Reply:
instead of comments please a an unit test which shows that the above JSON will be de-serialized into a proper DTO